Reorganize schema pipeline section (#619)
apollographql/docs: src/content/graphos/delivery/schema-checks.mdx

---
title: Schema checks

> subtitle: Make sure all proposed changes to your schema are safe

Reviewer: ```suggestion
subtitle: Validate all proposed changes to your schema are safe
```

   - [Learn how to publish schemas.](./)
2. For operation checks, make sure your supergraph is sending operation metrics to GraphOS. GraphOS uses historical metrics to determine whether a potentially dangerous schema change is safe.

>    <Caution>
> 
>    If GraphOS has no operation metrics to compare against, all potentially
>    dangerous schema changes result in a failed check!
> 
>    </Caution>

Reviewer: ```suggestion
   <Caution>

   If GraphOS has no operation metrics to compare against, all potentially dangerous schema changes result in a failed check.

   </Caution>
```

Let's say you've made local changes to one of your subgraph schemas. After you [complete the prerequisites](#prerequisites), you can run schema checks against those changes with the `rover subgraph check` command.

<Note>

> If you have a [monograph](../graphs/#monographs) (not recommended), use [`rover graph`](/rover/commands/graphs/#checking-schema-changes) commands instead of `rover subgraph` commands.

Reviewer: ```suggestion
If you have a [monograph](../graphs/#monographs) (which is not recommended), use [`rover graph`](/rover/commands/graphs/#checking-schema-changes) commands instead of `rover subgraph` commands.
```
